import-service/src/handlers/__test__/handlers.test.js

```diff
@@ -0,0 +1,27 @@
+import * as AWS from 'aws-sdk';
+import * as AWSMock from 'aws-sdk-mock';
+
+import { importProductsFile } from '../importProductsFile';
+import { S3_PUT_OPERATION } from '../../constants';
+import { getResponseObject, getS3ImportParams } from '../../utils';
+
+describe('AWS Lambda handlers tests', () => {
+  const testName = 'test.csv';
+  const getEventMock = name => ({
+    queryStringParameters: { name },
+  });
+
+  describe('importProductsFile handler tests', () => {
```

Comment:
no objections, just suggestion to add more tests for negative scenarios (400, 500 errors for example)

Reply:
Thank you for the review and your advices!